doc/dev/error-message.md

```diff
@@ -0,0 +1,14 @@
+# Error message
+
+Elektra has a lot of developers and hence the error messages vary in how they are written.
+This guideline should unify the message format so that users will get consistent error messages.
+
+- All error messages start with a capital letter even if you reference a method name such as "KdbSet() not supported". The only exception are quoted messages at the start such as "'xxxx-xxxx' is an invalid syntax". Try to avoid variable strings at the start of the sentence ("%s"-strings) if possible because you cannot guarantee capital casing.
```

Comment:
What about also quoting method names? Naming them wrongly in the error messages does not seem right.